tests/nav-spine_in-spine-no-list-style/OPS/package.opf

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Navigation Documents</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:creator>Dave Cramer</dc:creator>
+    <dc:date>2022-01-25</dc:date>
+    <dc:description>
+      The navigation document is also referenced in the spine; however, the TOC items in the Reading System's own control must not use the list style type of the navigation document.
+    </dc:description>
+    <dc:identifier id="pub-id">nav-spine_in-spine-no-list-style</dc:identifier>
```

Comment:
As below, I don't see a need for the nav to be in the spine in this test. I would rename this test to nav-list-style.

Reply:
See the discussion below.

Reply:
OK, but even so I think this needs a rename. The current style, nav-spine_something, means this test relates to a conformance requirement that *could* be called nav-spine, but we've already used that name for a separate requirement. Therefore this should still be nav-list-style, or nav-list-style_in-spine if you were going to have an in-spine/not-in-spine pair of tests (which seems unnecessary).